README.md

```diff
@@ -233,6 +233,7 @@ To check how debugging works we need to:
 - set up a breakpoint;
 - make a call to execute some functionality;
 
+### Debug for a couple components
```

Comment:
`for` is not needed here. It's debug _something_ (as on line 19)

Reply:
thanks, fixed